static/scss/partials/premium-promo.scss

```diff
@@ -0,0 +1,63 @@
+.premium-promo-hero-cta {
+    text-align: center;
+
+    @media screen and #{$mq-md} {
+        text-align: start;
+    }
+}
+
+
+.premium-promo-hero-cta button:disabled,
+.perk-cta:disabled {
+    cursor: not-allowed;
+    pointer-events: visible;
+}
+
+.premium-promo-availability-warning {
+    padding: $spacing-md 0;
+}
+
+.perks-headline {
+    @include text-title-sm;
+    text-align: center;
+}
+
+.perks-lead {
+    @include text-body-md;
+    display: block;
+
+    @media screen and #{$mq-md} {
+        margin: 0 auto;
+    }
+    width: 75%;
+}
+
+.perk {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-2xl;
+    align-items: center;
+    padding: $spacing-2xl 0;
+
+    @media screen and #{$mq-md} {
+        flex-direction: row;
+        padding: $spacing-2xl;
+
+        &:nth-child(odd) {
```

Comment:
Looking at the design, this should actually be `even`. The Figma file shows the first content section on desktop has the image on the left and the copy on the right, while the demo site has it flipped.

Reply:
Good eye, thanks for the fix!